src/datasources/erc1155.ts

```diff
@@ -146,6 +146,6 @@ export function handleURI(event: URIEvent): void
 {
 	let contract = fetchERC1155(event.address)
 	let token    = fetchERC1155Token(contract, event.params.id)
-	token.uri    = event.params.value
+	token.uri    = event.params.value.replaceAll('{id}', event.params.id.toString());
```

Comment:
According to the spec the id has to be hex-encoded and padded to 64 chararacters.

```suggestion
	token.uri    = event.params.value.replaceAll('{id}', event.params.id.toString(16).padStart(64, '0'));
```

Reply:
does it need to be 0x prefixed ?